src/locale/locales/ca/messages.po

```diff
@@ -603,12 +603,12 @@ msgid "Check your inbox for an email with the confirmation code to enter below:"
 msgstr "Comprova el teu correu per rebre el codi de confirmació i entra'l aquí sota:"
 
 #: src/view/com/modals/Threadgate.tsx:72
-msgid "Choose \"Everybody\" or \"Nobody\""
-msgstr "Tria \"Tothom\" or \"Ningú\""
+msgid "Choose \"Everybody\" or \"Nobody\
+msgstr "Tria \"Tothom\" or \"Ningú\
```

Comment:
I think there might be some quotation marks missing here

Reply:
Thanks! Corrected